pax/config/Simulation.ini

```diff
@@ -5,3 +5,1571 @@ look_for_config_in_runs_db = False
 input = 'WaveformSimulator.WaveformSimulatorFromCSV'
 decoder_plugin = None
 pre_dsp = 'ZLE.SoftwareZLE'
+
+[WaveformSimulator]
+# an example of PMT after pulses configuration for each PMT
+# the precise one for Xe1T will come later after analysis
+# this is just for making tester happy
+each_pmt_afterpulse_types = {
+    0:{
```

Comment:
Okay, I can modify this for the test. But this is just for showing the format. In future when Julien got the profile for each PMT, it needs to be put there like this.